internal/boxcli/init.go

```diff
@@ -7,26 +7,35 @@ import (
 	"github.com/pkg/errors"
 	"github.com/spf13/cobra"
 	"go.jetpack.io/devbox"
+	"go.jetpack.io/devbox/internal/boxcli/writer"
 )
 
+type initCmdFlags struct {
+	quiet bool
+}
+
 func InitCmd() *cobra.Command {
+	flags := initCmdFlags{}
 	command := &cobra.Command{
 		Use:   "init [<dir>]",
 		Short: "Initialize a directory as a devbox project",
 		Long:  "Initialize a directory as a devbox project. This will create an empty devbox.json in the current directory. You can then add packages using `devbox add`",
 		Args:  cobra.MaximumNArgs(1),
 		RunE: func(cmd *cobra.Command, args []string) error {
-			return runInitCmd(cmd, args)
+			return runInitCmd(cmd, args, flags)
 		},
 	}
+	command.Flags().BoolVarP(
+		&flags.quiet, "quiet", "q", false, "Quiet mode: Suppresses logs.")
```

Comment:
if we decide to not make a global flag, then instead of copy-pasta this flag declaration could we move it to a function that all the commands call?

Reply:
not all commands have to have `-q`. I prefer going with your 2nd suggestion.

Reply:
I ended up adding `-q` to the root command. I think it makes the most sense there. This way we can implement the quiet behaviour for future commands with minimal change.